remoting/zookeeper/client.go

```diff
@@ -392,8 +392,16 @@ func (z *ZookeeperClient) Close() {
 	logger.Warnf("zkClient{name:%s, zk addr:%s} exit now.", z.name, z.ZkAddrs)
 }
 
-// Create ...
+// Create will create the node recursively, which means that if the parent node is absent,
+// it will create parent node first.
+// And the value for the basePath is ""
 func (z *ZookeeperClient) Create(basePath string) error {
+	return z.CreateWithValue(basePath, []byte(""))
+}
+
+// Create will create the node recursively, which means that if the parent node is absent,
+// it will create parent node first.
```

Comment:
```suggestion
// CreateWithValue will create the node recursively, which means that if the parent node is absent,
// it will create parent node first.
```